src/components/ProgressBar.svelte

```diff
@@ -42,54 +42,65 @@
 </div>
 
 <style lang="scss">
-  $bg: #fff;
-  $text: #999;
-  $border: #e0e0e0;
-  $circle-active: #3498db;
+  $silver: #d3d3d3;
```

Comment:
not using all these, but they are copied from the tailwind config styles, so leaving here for a design pass.